db/migrate/20180131020132_drop_unique_github_id_index_on_organizations.rb

```diff
@@ -0,0 +1,6 @@
+class DropUniqueGitHubIdIndexOnOrganizations < ActiveRecord::Migration[5.1]
+  def change
+    remove_index :organizations, :github_id
+    add_index :organizations, :github_id
```

Comment:
Why are you removing and adding the index here?

Reply:
It's the only way I could drop the unique aspect of the index since it looks like that isn't a valid modification http://api.rubyonrails.org/classes/ActiveRecord/Migration.html

Reply:
Ohh, I thought the intent was to remove the index. Cool :+1: